### DISCUSSION TITLE
How to plot many illuminants with different scatter settings?

### DISCUSSION THREAD

**Discussion by LaughingMaann** [colour-science/colour#1006] [General]
how do i format the illuminants kwargs, if i want to plot something similar in xy? this dictionary of 3 keys each with corresponding xy values of same length, can be plotted in the xy chart, shown here. but when i try to use this same format to uv, there's an error
<img width="994" alt="image" src="https://user-images.githubusercontent.com/16868265/176411008-fdd4836d-41ad-4f98-be57-4de9e7d348ab.png">

**Comment by LaughingMaann** [comment]
this is the error when trying to plot that format of dictionary illuminants for uv

<img width="953" alt="image" src="https://user-images.githubusercontent.com/16868265/176411545-3227c0d5-ac25-473b-9815-b79f19fedafe.png">

**Comment by LaughingMaann** [comment]
the goal is to get something like this but in uv; however, i dont want to build a dictionary w/ len 10,026 or 9,345 keys. else the there's no way to differentiate the markers 


<img width="901" alt="image" src="https://user-images.githubusercontent.com/16868265/176412812-0f8e9717-2750-4b43-8df0-25c11a5db00a.png">

**Comment by LaughingMaann** [comment]
@KelSolaar senpai

**Comment by thomasmansencal** [comment]
As you need such a large quantity of points, I would recommend three scatter plots on top of the diagram that you want:

```python
import colour.plotting
import numpy as np

figure, axes = colour.plotting.plot_chromaticity_diagram_CIE1976UCS(standalone=False)
colour.plotting.temperature.plot_planckian_locus(method="CIE 1976 UCS", axes=axes, standalone=False)
axes.scatter(np.random.random(100), np.random.random(100), color="r", marker="x")
axes.scatter(np.random.random(100), np.random.random(100), color="g", marker="v")
axes.scatter(np.random.random(100), np.random.random(100), color="b", marker="+")
colour.plotting.render()
```

![image](https://user-images.githubusercontent.com/99779/176618357-bf67b9ae-4db0-48e1-a6d5-b61306b6df59.png)

**Reply by LaughingMaann** [reply]
^wait sorry i get this as an error when just running that above example

<img width="1341" alt="image" src="https://user-images.githubusercontent.com/16868265/176877883-61e9d261-e66c-4061-b875-9df467422275.png">

**Reply by LaughingMaann** [reply]
senpai, your example works when i do this instead
<img width="903" alt="image" src="https://user-images.githubusercontent.com/16868265/176878913-e0f2abed-d25f-4909-a89d-d8242c4537a3.png">

**Reply by LaughingMaann** [reply]
OMG YAS I GOT IT THANKS
<img width="627" alt="image" src="https://user-images.githubusercontent.com/16868265/176879006-39417ce5-e510-4d34-9bb0-98560751afb3.png">

**Comment by LaughingMaann** [comment]
@KelSolaar is there a way to set the markersize in your provided example? i cannot seem to set one

**Reply by thomasmansencal** [reply]
The parameter should be `s`: https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.scatter.html
